Controller/ContentTypeController.php

```diff
@@ -127,6 +140,8 @@ public function viewContentTypeGroupAction($contentTypeGroupId)
             )->createView();
 
             $canDeleteById[$contentTypeId] = $canDelete && !$this->contentTypeService->isContentTypeUsed($contentType);
+
+            $languageById[$contentTypeId] = $this->getVerifiedContentTypeLanguageCode($contentType);
```

Comment:
Thanks, sounds good. (Only I'd use `$contentType->mainLanguageCode`) instead of `$languageCodes[0]` as I'm not sure that the first language is the main one.)